modules/ebi-metagenomics/generategaf/main.nf

```diff
@@ -0,0 +1,44 @@
+
+process GENERATEGAF {
```

Comment:
is it allowed to use _ in names in nf-core standards? I think better separate words for understanding..

Reply:
not allowed unfortunately, `nf-core modules create` yells at you if you try to include `_` in module names